src/components/pages/MapView.tsx

```diff
@@ -14,8 +15,12 @@ function MapView() {
         if (webcam === null) {
             return null;
         }
+        let rotationAngle = 0;
+        if (webcam.direction !== undefined) {
+            rotationAngle = webcam.direction - 90; // unrotated icon points east
+        }
```

Comment:
```suggestion
        const rotationAngle = webcam.direction === undefined ? 0 : webcam.direction - 90
```

Reply:
I would consider the longer version more readable but it's your project.